sdk/file_sdk/Common/Common.pro

```diff
@@ -27,5 +30,10 @@ HEADERS += \
     istream.h \
     std_stream_adapter.h \
     exceptions.h \
-    string_utils.h
+    string_utils.h \
+    gsf_input_istream.h
 
+unix {
```

Comment:
unix!mac

Reply:
libgsf support already in. and it is entered like this.

---
In reply to: [139411427](https://github.com/AzureAD/rms-sdk-for-cpp/pull/158#discussion_r139411427) [](ancestors = 139411427)